cmake/valgrind-dlite.supp

```diff
@@ -220,3 +220,14 @@
    fun:expand_dynamic_string_token
    fun:_dl_map_object
 }
+
+# For Pydantic
+
+{
+   <insert_a_suppression_name_here>
```

Comment:
Please insert a suppression name here.

Reply:
Added comment and moved suppression to valgrind-python.supp.